libraries/AP_RPM/AP_RPM.h

```diff
@@ -18,6 +18,7 @@
 #include <AP_HAL/AP_HAL.h>
 #include <AP_Param/AP_Param.h>
 #include <AP_Math/AP_Math.h>
+#include <AP_InertialSensor/AP_InertialSensor.h>
```

Comment:
this could be put only in RPM_HarmonicNotch.cpp, can't it ?

Reply:
Thanks, fixed